pkg/config/load.go

```diff
@@ -426,3 +433,13 @@ func LoadByOrgRepo(path string) (ByOrgRepo, error) {
 	}
 	return config, nil
 }
+
+func (p *Prowgen) SkipPresubmits(branch string, variant string) bool {
+	if p.SkipOperatorPresubmits.Branch == "" && p.SkipOperatorPresubmits.Variant == "" {
+		return false
+	}
+	if (p.SkipOperatorPresubmits.Branch != "*" && p.SkipOperatorPresubmits.Branch != branch) || p.SkipOperatorPresubmits.Variant != variant {
```

Comment:
This is not a proper way to support the asterisk to indicate all branches. You will need to support regexes which is not that important for this change. Let's just remove it. Also, both statements return false, which is technically one statement.

Reply:
```go
func (p *Prowgen) SkipPresubmits(branch string, variant string) bool {
    return p.SkipOperatorPresubmits.Branch == branch && p.SkipOperatorPresubmits.Variant == variant
```